code/game/objects/items/food/meat.dm

```diff
@@ -846,6 +846,10 @@
 	name = "monkey meat"
 	foodtypes = RAW | MEAT
 
+/obj/item/food/meat/slab/bugmeat
```

Comment:
why not give it the spider meat icon state?

Reply:
great suggestion